components/deleteButton.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { Button, Container, Form, InputField, Row, Spacer } from "@yakad/ui";
+import { useRouter } from "next/navigation";
+import { useFetch, useFormDataHandle } from "@yakad/lib";
+import React from "react";
+
+interface Delete {
+    controller: string;
+    uuid: string;
+}
+
+export default function DeleteButton() {
+    const router = useRouter();
+    const [formData, setFormData] = React.useState<Delete>();
+
+    const fetch = useFetch(
+        `${process.env.NEXT_PUBLIC_API_URL}/?{controller/?{uuid}`,
+        {
+            method: "DELETE",
+            headers: {
+                Accept: "application/json",
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(formData),
+        }
+    );
+
+    const handler = useFormDataHandle(setFormData);
+    function deleted() {
+        if (confirm("Are you sure to delete a uuid") == true) {
```

Comment:
`if (confirm("Are you sure to delete UUID:" + props.uuid) == true) {`

or:

`if (confirm("Are you sure to delete: " + props.name ? props.name : props.uuid) == true) {`